test/IRGen/round-trip-debug-types-typealias.swift

```diff
@@ -8,3 +8,5 @@
 protocol Protocol<T> { associatedtype T }
 typealias AnyProtocol<T> = any Protocol<T>
 let crash: AnyProtocol<Any?>
+
+// CHECK: @"$s4main5crashAA8Protocol_pypSg1TAaCPRts_XPvp"
```

Comment:
It would be nice to include a little more context here such which !DI...() node this appears inside of.

Reply:
Done. PTAL